aten/src/ATen/function_wrapper.py

```diff
@@ -1690,7 +1690,7 @@ def emit_body(env, option, scalar_type_cases):
                     arguments_indices = ret['arguments']
                     arguments = [option['arguments'][argi]
                                  for argi in arguments_indices]
-                    if scalar_check is not None:
+                    if scalar_check is not None and scalar_check != 'false':
```

Comment:
This makes the check that you previously added (l1702 in this file) inside irrelevant no?

Reply:
I don't think so?  I can codegen to check, but it's possible that the scalar_check is a dict but an individual scalar_check is false.

Reply:
Looks like what I said was accurate: https://gist.github.com/gchanan/75801c24c4201656dbdb9615697b6250